cypress/e2e/read-receipts/high-level.spec.ts

```diff
@@ -356,6 +422,16 @@ describe("Read receipts", () => {
                 saveAndReload();
                 assertRead(room2);
             });
+            it.skip("Sending a message from a different client marks room as read", () => {
```

Comment:
Is this supposed to be `skip`? (If it fails, please add a comment saying why.)

Reply:
Why "from a different client" here? `sendMessages` sends from our normal Cypress client (right?).